charts/stable/kopia/values.yaml

```diff
@@ -7,13 +7,17 @@ service:
   main:
     ports:
       main:
-        protocol: http
         port: 10238
+    protocol: http
+    certs:
+      generate: false
+      certPath: /app/config/kopia.cert
+      keyPath: /app/config/kopia.key
```

Comment:
This is not an extention that we allow from common at all and wont even work as indent is wrong